### PR TITLE
[MSGINA] gui: Improve DlgData_LoadBitmaps()

### PR DESCRIPTION
**Pull request by SergeGautherie**
And let bar timer depend on the bar image only.

### CODE REVIEW

**Comment by SergeGautherie** [comment]
Addendum to 623dd26ccea5e79001aee769c230fb4f8e304fc8: Cc @katahiromz

File: dll/win32/msgina/gui.c [hunk 1]
```diff
@@ -56,28 +56,34 @@ DlgData_Create(HWND hwndDlg, PGINA_CONTEXT pgContext)
     return pDlgData;
 }
 
-static BOOL
-DlgData_LoadBitmaps(PDLG_DATA pDlgData)
+static void
```
Comment (Fraizeraust) [inline dll/win32/msgina/gui.c:59]: How would declaring the function as VOID be considered as an improvement? Could you elaborate?

Comment (SergeGautherie) [reply dll/win32/msgina/gui.c]: Case fixed.
The current return value is not what is wanted, then it becomes unused.

File: dll/win32/msgina/gui.c [hunk 2]
```diff
@@ -56,28 +56,34 @@ DlgData_Create(HWND hwndDlg, PGINA_CONTEXT pgContext)
     return pDlgData;
 }
 
-static BOOL
-DlgData_LoadBitmaps(PDLG_DATA pDlgData)
+static VOID
+DlgData_LoadBitmaps(_Out_ PDLG_DATA pDlgData)
```
Comment (learn-more) [inline dll/win32/msgina/gui.c:60]: Technically this is `_Inout_`

Comment (SergeGautherie) [reply dll/win32/msgina/gui.c]: Right, I missed to account for the `pDlgData->pgContext->hDllInstance` read. Fixed.

**Pull request merged**